drupal.make

```diff
@@ -3,17 +3,15 @@ api = 2
 
 ; Core
 projects[drupal][type] = core
-projects[drupal][version] = 7.60
-projects[drupal][patch][] = "http://drupal.org/files/issues/menu-get-item-rebuild-1232346-45.patch"
-projects[drupal][patch][] = "http://drupal.org/files/ssl-socket-transports-1879970-13.patch"
```

Comment:
This patch has been removed but as far as I can tell the corresponding issue has not been resolved. Is this intentional?

Reply:
It was an patch I crated back in the time to fix issues with certificates and ALMA. But it have been resolved in other ways and is not an issue for the current setup of DDB CMS services.